src/containers/layout/app-snackbar/AppSnackbar.tsx

```diff
@@ -0,0 +1,36 @@
+import { useAppDispatch, useAppSelector } from '~/hooks/use-redux'
+import Snackbar from '@mui/material/Snackbar'
+import Alert from '@mui/material/Alert'
+import { Box } from '@mui/material'
```

Comment:
Use default omport here

Reply:
Thanks, it was replaced with default import.